src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/ProfessionalReferenceDataClient.java

```diff
@@ -1022,6 +1023,31 @@ public Map<String, Object> findOrganisationsByUserId(String userId, String role)
         return getResponse(responseEntity);
     }
 
+
+
+    public Map<String, Object> updateOrgName(
+        OrganisationNameUpdateRequest organisationNameUpdateRequest, String role, String organisationIdentifier) {
+
+        ResponseEntity<Map> responseEntity = null;
+        String urlPath = "http://localhost:" + prdApiPort + APP_INT_BASE_PATH + "/" + organisationIdentifier + "/name";
+        try {
+            HttpEntity<OrganisationNameUpdateRequest> requestEntity = new HttpEntity<>(organisationNameUpdateRequest,
+                getMultipleAuthHeaders(role));
+            responseEntity = restTemplate.exchange(urlPath, HttpMethod.PUT, requestEntity, Map.class);
+
+        } catch (RestClientResponseException ex) {
+            HashMap<String, Object> statusAndBody = new HashMap<>(2);
```

Comment:
minor: map creation could be replaced by using Map.of(k1, v1, k2, v2) factory method